Makefile

```diff
@@ -395,6 +374,18 @@ define buildpage
 		--var "staging"="$(DEPLOY_TARGET)" $< > $@
 endef
 
+define applypatches
+	git apply --directory=src scripts/fastclick.patch;
+	git apply --directory=src/lib scripts/slipjs.patch;
```

Comment:
I would form the patches in the same way (look at the directory that is different)